generator/templates/server/parameter.gotmpl

```diff
@@ -173,7 +173,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}Params) BindRequest(r *http.Requ
         break
       }
     }
-    {{ else }}if err := body.Validate(route.Formats); err != nil {
+    {{ else if or .IsComplexObject .IsTuple }}if err := body.Validate(route.Formats); err != nil {
```

Comment:
I think if you make this be: `not (or .IsInterface .IsPrimitive .IsCustomFormatter)`  then you don't need to edit the operation.go file